templates/rhpam74-authoring-ha-datagrid.yaml

```diff
@@ -271,25 +271,25 @@ parameters:
   value: 1Gi
   required: true
 ## OpenShift Enhancement BEGIN
-- displayName: Enable KIE server global discovery [Tech Preview]
+- displayName: Enable KIE server global discovery 
   description: "If set to true, turns on KIE server global discovery feature (Sets the org.kie.server.controller.openshift.global.discovery.enabled system property)"
   name: KIE_SERVER_CONTROLLER_OPENSHIFT_GLOBAL_DISCOVERY_ENABLED
   value: "false"
   required: false
-- displayName: Prefer KIE Server OpenShift Service [Tech Preview]
+- displayName: Prefer KIE Server OpenShift Service 
   description: Enables connection to KIE Server via OpenShift internal Service endpoint (Sets the org.kie.server.controller.openshift.prefer.kieserver.service system property)
   name: KIE_SERVER_CONTROLLER_OPENSHIFT_PREFER_KIESERVER_SERVICE
   value: "false"
   required: false
-- displayName: KIE ServerTemplate Cache TTL [Tech Preview]
+- displayName: KIE ServerTemplate Cache TTL 
   description: KIE ServerTemplate Cache TTL in milliseconds (Sets the org.kie.server.controller.template.cache.ttl system property)
   name: KIE_SERVER_CONTROLLER_TEMPLATE_CACHE_TTL
   value: "60000"
   required: false
-- displayName: Enable OpenShift Integration [Tech Preview]
+- displayName: Enable OpenShift Integration 
   description: If set to true, turns on OpenShift integration feature (Sets the org.kie.workbench.controller.openshift.enabled system property)
   name: KIE_WORKBENCH_CONTROLLER_OPENSHIFT_ENABLED
-  value: "false"
+  value: "true"
```

Comment:
If we are taking away the ability to configure a controller, and only allowing for OSSS, why make KIE_WORKBENCH_CONTROLLER_OPENSHIFT_ENABLED here configurable as a template parameter? If it's not really an option, then we should hardcode it in the rhpamcentr DeyploymentConfig.

Reply:
For 7.4 release, our goal is to set the default startup strategy to OSSS, rather than completely remove 'Controller'. It should be the case in future release.

Reply:
Same response as above.

Reply:
For templates contain BC/WB component, removed above mention template parameter and explicitly set the value of related environment variable to 'true'.